bin/run-tests-in-docker.sh

```diff
@@ -16,9 +16,9 @@
 docker build --rm -t exercism/groovy-test-runner .
 
 # Run the Docker image using the settings mimicking the production environment
+# --read-only cannot be used because it causes https://github.com/gradle/gradle/issues/8107
```

Comment:
@iHiD is this going to be a problem in prod?

Reply:
@angelikatyborska Not yet :)